[2024-02-12T17:00:03.026+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.weather_api_ready scheduled__2024-02-12T16:30:00+00:00 [queued]>
[2024-02-12T17:00:03.039+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.weather_api_ready scheduled__2024-02-12T16:30:00+00:00 [queued]>
[2024-02-12T17:00:03.040+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-02-12T17:00:03.060+0000] {taskinstance.py:2191} INFO - Executing <Task(HttpSensor): weather_api_ready> on 2024-02-12 16:30:00+00:00
[2024-02-12T17:00:03.069+0000] {standard_task_runner.py:60} INFO - Started process 345 to run task
[2024-02-12T17:00:03.194+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'weather_api_ready', 'scheduled__2024-02-12T16:30:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/Weather_dag.py', '--cfg-path', '/tmp/tmpzg4xo_84']
[2024-02-12T17:00:03.221+0000] {standard_task_runner.py:88} INFO - Job 9: Subtask weather_api_ready
[2024-02-12T17:00:03.528+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dag.weather_api_ready scheduled__2024-02-12T16:30:00+00:00 [running]> on host 07696bf07ae2
[2024-02-12T17:00:03.760+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='stellar' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='weather_api_ready' AIRFLOW_CTX_EXECUTION_DATE='2024-02-12T16:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-12T16:30:00+00:00'
[2024-02-12T17:00:03.762+0000] {http.py:130} INFO - Poking: /data/2.5/weather?q=Bangkok&appid=d69585a88935d2a759a90a77d406e260
[2024-02-12T17:00:03.783+0000] {base.py:83} INFO - Using connection ID 'weather_conn_id' for task execution.
[2024-02-12T17:00:04.148+0000] {base.py:295} INFO - Success criteria met. Exiting.
[2024-02-12T17:00:04.163+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=weather_api_ready, execution_date=20240212T163000, start_date=20240212T170003, end_date=20240212T170004
[2024-02-12T17:00:04.212+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-12T17:00:04.254+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-02-12T17:12:38.839+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.weather_api_ready scheduled__2024-02-12T16:30:00+00:00 [queued]>
[2024-02-12T17:12:38.869+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.weather_api_ready scheduled__2024-02-12T16:30:00+00:00 [queued]>
[2024-02-12T17:12:38.870+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-02-12T17:12:38.911+0000] {taskinstance.py:2191} INFO - Executing <Task(HttpSensor): weather_api_ready> on 2024-02-12 16:30:00+00:00
[2024-02-12T17:12:38.934+0000] {standard_task_runner.py:60} INFO - Started process 104 to run task
[2024-02-12T17:12:38.946+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'weather_api_ready', 'scheduled__2024-02-12T16:30:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/Weather_dag.py', '--cfg-path', '/tmp/tmpobuzhnjd']
[2024-02-12T17:12:38.953+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask weather_api_ready
[2024-02-12T17:12:39.095+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dag.weather_api_ready scheduled__2024-02-12T16:30:00+00:00 [running]> on host 5038fee5e684
[2024-02-12T17:12:39.263+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='stellar' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='weather_api_ready' AIRFLOW_CTX_EXECUTION_DATE='2024-02-12T16:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-12T16:30:00+00:00'
[2024-02-12T17:12:39.266+0000] {http.py:130} INFO - Poking: /data/2.5/weather?q=Bangkok&appid=d69585a88935d2a759a90a77d406e260
[2024-02-12T17:12:39.282+0000] {base.py:83} INFO - Using connection ID 'weather_conn_id' for task execution.
[2024-02-12T17:12:39.624+0000] {base.py:295} INFO - Success criteria met. Exiting.
[2024-02-12T17:12:39.646+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=weather_api_ready, execution_date=20240212T163000, start_date=20240212T171238, end_date=20240212T171239
[2024-02-12T17:12:39.721+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-12T17:12:39.773+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
